docs/section-2-using-parsers.md

```diff
@@ -651,7 +666,79 @@ And this pattern would match key-value pairs where the `value` is an identifier
 )
 ```
 
-_Note_ - Predicates are not handled directly by the Tree-sitter C library. They are just exposed in a structured form so that higher-level code can perform the filtering. However, higher-level bindings to Tree-sitter like [the Rust crate](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_rust) or the [WebAssembly binding](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_web) implement a few common predicates like `#eq?` and `#match?`.
+The prefix "any-" and "all-" are meant for use with quantified captures. Here's
+an example finding a segment of empty comments
+
+```scheme
+((comment)+ @comment.empty
+  (#any-eq? @comment.empty "//"))
+```
+
+Note that any-eq will match a quantified capture if
+_any_ of the nodes match the predicate, while all-eq will only match if
+_all_ the nodes match the predicate.
+
+#### match?, not-match?, any-match?, any-not-match?, all-match?, all-not-match?:
+
+These predicates are similar to the eq? predicates, but they use regular expressions
+to match against the capture's text.
+
+The first argument must be a capture, and the second must be a string containing
+a regular expression.
+
+For example, this pattern would match identifier whose name is written in `SCREAMING_SNAKE_CASE`:
+
+```scheme
+((identifier) @constant
+  (#match? @constant "^[A-Z][A-Z_]+"))
+```
+
+Again, the prefix "any-" and "all-" are meant for use with quantified captures.
+
+Here's an example finding potential documentation comments in C
+
+```scm
```

Comment:
No editors know the `scm` tag, use the `scheme` tag instead everywhere like you did above.

Reply:
Well, GitHub should create it -- after all, there's a tree-sitter parser for it ;)

Reply:
We using the `scheme` tag not for Github, we use it for a renderer used in docs.